srcpkgs/volctl/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'volctl'
 pkgname=volctl
-version=0.8.2
+version=0.9.0
 revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-gobject gtk+3 desktop-file-utils python3-cairo python3-yaml python3-click"
-checkdepends="${depends} dbus pulseaudio xvfb-run"
+checkdepends="${depends} dbus pulseaudio xvfb-run python3-wheel python3-pip"
```

Comment:
they don't have tests, why is this here? can we also remove do_check, because it doesn't do anything?

Reply:
https://github.com/void-linux/void-packages/pull/28806#discussion_r578131916
This is the reason.
I am open to delete it.
Ping @ericonr

Reply:
Feel free to remove, ahesford's complained about me recommending adding checkdepends when tests aren't defined too.

Reply:
If make_check=extended is correct, this can be marked as resolved. If I should use make_check=no, let me know.

Reply:
I suppose it'd be `make_check=no`, since there aren't any to be run.

Reply:
You should use `make_check=no # no tests, but check fails with a non-zero return code` or something similar

Reply:
oops, I was a few seconds late. Anyway, a comment with the reason would be nice.

Reply:
@paper42 your are right about a comment. Added.